index.bs

```diff
@@ -843,6 +845,12 @@ An attribution source is a [=struct=] with the following items:
 :: Number of [=aggregatable debug reports=] created for this [=attribution source=].
 : <dfn>aggregatable debug reporting config</dfn>
 :: An [=aggregatable debug reporting config=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer representing the number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
```

Comment:
Does this field have to be nullable, or can we provide a suitable default?

Reply:
@giladbarkan-github @vikassahu29 and I have [discussed](https://googleplex-android-review.git.corp.google.com/c/platform/packages/modules/AdServices/+/26663673/comment/12b420b2_25c6e298/) this before and the conclusion this needs to be nullable to be backward compatible.

TLDR:
If we default it to 1 (so that max information gain doesn't change): 
1. attribution_scope_limit = 1 should be a valid use case
2. Unexpected side effects if we default the limit to 1: if a new source registered with higher attribution scope limit > 1, all older non scoped source would be removed as it now has smaller attribution scope limit. In other words we need to differentiate attribution_scope_limit = 1 and not set.